cmd/generator/main.go

```diff
@@ -105,7 +111,9 @@ func main() { // nolint:gocyclo
 		for _, name := range keys {
 			// We don't want Aws prefix in all kinds.
 			kind := strings.TrimPrefix(strcase.ToCamel(name), "Aws")
-			if err := crdGen.Generate(version, kind, resources[name]); err != nil {
+			resource := resources[name]
+			resource.Schema["region"] = regionSchema
```

Comment:
```suggestion
			resources[name].Schema["region"] = regionSchema
```

Since it's not used anywhere else.

Reply:
Actually, `resource` (`resources[name]`) is used in the line below.

Reply:
Yeah it's a single usage and very close where it's used, so another variable is [usually](https://github.com/golang/go/wiki/CodeReviewComments#variable-names) not necessary in such cases. Even if it is initialized, it'd have a very short name.

Reply:
I am not aware of a convention like if it has two usages, **usually** a new variable is not needed :) The link you shared points to short variable-length if closer, which I would agree with in general.